weekly-updates/next.md

```diff
@@ -2,39 +2,35 @@
 
 Hello and welcome to another issue of This Week in Solid!
 
-[Solid](https://solid.github.io/information/) is a standard that describes how to build data storage and applications in such a way that users can conveniently switch between data storage providers and application providers and take the data generated along.
```

Comment:
Generally I think these changes are good, because it is indeed hard to find updates information in here. That said, I do think one line (i.e. this one) that describes the project at a high level would be good for people encountering an issue of TWiS for the first time - perhaps linking to more detailed information.

See for instance [This Week in Rust](https://this-week-in-rust.org/blog/2019/08/20/this-week-in-rust-300/): there's a short paragraph for newcomers that's easy and natural to skip if you're familiar with the format, but provides some essential context for newcomers.